hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/rpc/RpcClient.java

```diff
@@ -441,6 +431,8 @@ public void createBucket(
     verifyVolumeName(volumeName);
     verifyBucketName(bucketName);
     Preconditions.checkNotNull(bucketArgs);
+    verifyCountsQuota(bucketArgs.getQuotaInCounts());
+    verifySpaceQuota(bucketArgs.getQuotaInBytes());
```

Comment:
`createBucket` verifies quota args, but `createVolume` does not.  Is this intentional?

Reply:
The PR of createVolume did not add a checkmark before, I will add this in the createVolume.